python/sidecar/storage.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from sidecar import typing
+
+
+class JsonStorage:
+    def __init__(self, filepath: str):
+        self.filepath = filepath
+        self.references = []
+        self.corpus = []
+        self.tokenized_corpus = []
+
+    def load(self):
+        with open(self.filepath, 'r') as f:
+            data = json.load(f)
```

Comment:
Should we have a try except here, in case the filepath or the file content is incorrect?